_episodes_rmd/02-starting-with-data.Rmd

```diff
@@ -137,81 +137,35 @@ The output tells us that is a data frame. Think of this structure as a spreadshe
 Data frames are very useful for storing data and you will use them frequently when programming in R.
 A typical data frame of experimental data contains individual observations in rows and variables in columns.
 
-What if we need the maximum inflammation for all patients, or the average for each day?
-As the diagram below shows, we want to perform the operation across a margin of the data frame:
-
-<img src="../fig/r-operations-across-margins.svg" alt="Operations Across Margins" />
-
-To support this, we can use the `apply` function.
-
-> ## Getting Help
->
-> To learn about a function in R, e.g. `apply`, we can read its help
-> documention by running `help(apply)` or `?apply`.
-{: .callout}
-
-`apply` allows us to repeat a function on all of the rows (`MARGIN = 1`) or columns (`MARGIN = 2`) of a data frame.
-
-Thus, to obtain the average inflammation of each patient we will need to calculate the mean of all of the rows (`MARGIN = 1`) of the data frame.
+What if we need the average inflammation of each patient, or the average for each day?
 
 ```{r}
-avg_patient_inflammation <- apply(dat, 1, mean)
+avg_patient_inflammation <- rowMeans(dat)
```

Comment:
Nice. Good idea. Not sure why they use apply in the original lesson...

Reply:
[Hasn't changed in years](https://github.com/swcarpentry/r-novice-inflammation/blame/gh-pages/_episodes_rmd/01-starting-with-data.Rmd#L397-L418), but we since there is [the idea of dropping `for` loops in favor of `purrr::map()`](https://github.com/swcarpentry/r-novice-inflammation/issues/250), we could extend that update to also remove apply.